conversation/go/http/README.md

```diff
@@ -23,6 +23,9 @@ name: Run multi app run template
 expected_stdout_lines:
   - '== APP - conversation == Input sent: What is dapr?'
   - '== APP - conversation == Output response: What is dapr?'
+  - '== APP - conversation == Tool calling input sent: What is the weather like in San Francisco in celsius?'
+  - '== APP - conversation == Output message: What is the weather like in San Francisco in celsius?'
+  - '== APP - conversation == Tool calls detected:'
```

Comment:
LGTM! We can use this as baseline for testing the tool calling now @sicoyle @ConstantinChirila